src/model/attendee.py

```diff
@@ -32,51 +34,53 @@ def __init__(self, id, course, year, gender, status, email, passHash):
         self.status = status
         self.email = email
         self.passHash = passHash
+
     def authenticate(self, password):
```

Comment:
This `authenticate` shit can probably be removed— just compare the hashes directly in the handler.

Reply:
We also won't need to authenticate `Attendees` anymore because `Users` are the ones who will be logging on from now on